Source/WebCore/html/HTMLFormControlElement.cpp

```diff
@@ -275,8 +275,12 @@ void HTMLFormControlElement::dispatchBlurEvent(RefPtr<Element>&& newFocusedEleme
 
 bool HTMLFormControlElement::shouldAutocorrect() const
 {
+    if (RefPtr input = dynamicDowncast<HTMLInputElement>(*this); input
+        && (input->isPasswordField() || input->isEmailField() || input->isURLField())) {
+        return false;
+    }
```

Comment:
Nit - no `{}` around if statements as well (the initializer clause probably throws off the style checker).

```suggestion
        && (input->isPasswordField() || input->isEmailField() || input->isURLField()))
        return false;
```

Reply:
I think they are needed here because the conditional itself is multi-line? Though surprisingly https://webkit.org/code-style-guidelines/#braces doesn't cover this.

Reply:
Hm.. I've interpreted the "single line" in style guidelines as referring to whether or not the body of the if statement, rather than the conditional, is multiple lines. But I agree it's sort of ambiguous (which kind of makes sense given that the rules here predate initializer if). We can just keep it as is.